Mathlib/Algebra/Order/BigOperators/GroupWithZero/Multiset.lean

```diff
@@ -0,0 +1,51 @@
+/-
+Copyright (c) 2024 Daniel Weber. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Daniel Weber
```

Comment:
`Multiset.prod_nonneg` dates from https://github.com/leanprover-community/mathlib3/pull/8787

Reply:
Sorry about that, is this better?

Reply:
Looks better, but you credited the wrong person for this file! The other file is good

Reply:
Oops